ubcpi/static/css/ubcpi.css

```diff
@@ -349,17 +479,66 @@ textarea.pi-options {
 }
 
 .correct-answer-option {
-    margin-bottom: 20px;
+    margin-top: 20px;
+    margin-bottom: 5px;
+}
+
+.show-correct {
```

Comment:
Darn it, good catch. I've just made a PR to our own repo to fix up these potential clashes. We'll get it into this merged PR asap. Thanks for pointing it out.